components/bpmn/bpmn-explorer-web/src/web/controller/advancedFilterController.jag

```diff
@@ -0,0 +1,45 @@
+<!--
+ ~ Copyright (c) 2015 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~      http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+-->
+<%
+include("/model/common.jag");
+
+if (request.isSecure()) {//check whether the request is secure or not
+
+	var authSuccess = session.get("authSuccess");
```

Comment:
How this works in a cluster? If multiple nodes serving requests for same user, it would ask user to login multiple times, since "authSuccess" is stored in the node's session.